### DISCUSSION TITLE
5 GitHub Tips you didn't know about | GitHub Secrets 01

### DISCUSSION THREAD

**Discussion by ghost** [EddieHubCommunity/support#1452] [Show and tell] [answered]
Just to shout out to this amazing video by Eddie Jaoude. It is the best video on GitHub I watched yet!

https://www.youtube.com/watch?v=LV-KRIS1AjQ

**Comment by eddiejaoude** [comment]
Thank you Krish!

**Comment by ghost** [comment, marked as answer]
Up is the answers :up:
